### PR TITLE
Fix event.listen callback to be called with two parameters

### PR DESCRIPTION
**Pull request by timarandras**
In Events Tutorial. To match the documentation in https://deepstream.io/docs/client-js/event/

### CODE REVIEW

File: content/docs/40-client-js/20-event/index.md [hunk 1]
```diff
@@ -59,17 +59,16 @@ client.event.emit('notifications', 'Maria just came online')
 |Argument|Type|Optional|Description|
 |---|---|---|---|
 |pattern|String (regex)|false|The pattern to match events which subscription status you want to be informed of|
-|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, (Boolean) isSubscribed, and response (Object)|
+|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, and response (Object)|
```
Comment (yasserf) [inline content/docs/40-client-js/20-event/index.md]: >  to or is no longer subscribed

Function does not get called, the onStop function does.

File: content/docs/40-client-js/80-datasync-client-record/index.md [hunk 2]
```diff
@@ -153,17 +153,16 @@ client.record.setData('user/homer', 'son', 'Bart', (err) => {
 |Argument|Type|Optional|Description|
 |---|---|---|---|
 |pattern|String (regex)|false|The pattern to match records which subscription status you want to be informed of|
-|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, (Boolean) isSubscribed and response (Object).|
+|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, and response (Object).|
```
Comment (yasserf) [inline content/docs/40-client-js/80-datasync-client-record/index.md]: > to or is no longer subscribed

**Review by yasserf — CHANGES_REQUESTED** [review]
Incorrect statement saying it gets called when subscribed/unsubscribed

File: content/docs/40-client-js/20-event/index.md (hunk 1 above)
Comment (timarandras) [reply content/docs/40-client-js/20-event/index.md]: Committed a fixed phrasing.

**Comment by yasserf** [comment]
Awesome thank you!

**Pull request merged**